src/common/general/iconPathUtil.ts

```diff
@@ -0,0 +1,42 @@
+import * as vscode from "vscode";
+import * as path from "path";
+import ContextProvider from "../vscode/extension/ContextProvider";
+
+const DEFAULT_LIGHT_RESOURCES_FOLDER_PATH = "resources/light";
+const DEFAULT_DARK_RESOURCES_FOLDER_PATH = "resources/dark";
+
+enum IconTheme {
+    Light,
+    Dark
+}
+
+type ThemedIconUris = {
+    light: vscode.Uri;
+    dark: vscode.Uri;
+};
+
+function getIconPath(fileName: string, theme: IconTheme): string {
+    const folderName = theme === IconTheme.Light
+        ? DEFAULT_LIGHT_RESOURCES_FOLDER_PATH
+        : DEFAULT_DARK_RESOURCES_FOLDER_PATH;
+
+    return ContextProvider.get().asAbsolutePath(path.join(folderName, fileName));
+}
+
+function getIconUri(fileName: string, theme: IconTheme): vscode.Uri {
+    return vscode.Uri.file(getIconPath(fileName, theme));
+}
+
+function getThemedIconUris(fileName: string): ThemedIconUris {
+    return {
+        light: getIconUri(fileName, IconTheme.Light),
+        dark: getIconUri(fileName, IconTheme.Dark)
+    };
+}
+
+export {
+    IconTheme,
+    getIconPath,
+    getIconUri,
+    getThemedIconUris
```

Comment:
I think we should export the only function needed here. Exposing others might be confusing. Only exporting `getThemedIconUris` might force users to choose a theme for their icons.

Reply:
There are places where theme is not used. `getIconPath` is used outside this file. I also exported getIconUri for a possible future use case.